### PR TITLE
adding a helm chart

### PR DESCRIPTION
**Pull request by thomasjungblut**

### CODE REVIEW

File: charts/nvidia-installer/templates/ds-k8s-nvidia-deviceplugin.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      priorityClassName: system-node-critical
+      volumes:
+      - name: device-plugin
+        hostPath:
+          path: {{ required "nvidia-device-plugin.hostDevicePluginPath" .Values.nvidiaDevicePlugin.hostDevicePluginPath }}
+      - name: dev
+        hostPath:
+          path: /dev
+      containers:
+      - image: "k8s.gcr.io/nvidia-gpu-device-plugin@sha256:0842734032018be107fa2490c98156992911e3e1f2a21e059ff0105b07dd8e9e"
```
Comment (afritzler) [inline charts/nvidia-installer/templates/ds-k8s-nvidia-deviceplugin.yaml]: I would make the images also configurable via the `values.yaml` and set this one as default there.

File: charts/nvidia-installer/templates/ds-k8s-nvidia-driver.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+spec:
+  template:
+    metadata:
+      labels:
+        name: {{ template "nvidia-installer.name" . }}
+        k8s-app: {{ template "nvidia-installer.name" . }}
+    spec:
+      hostPID: true
+      initContainers:
+      - image: squat/modulus:5c0e39827e98671928b1742d3d194860ed0345e8
```
Comment (afritzler) [inline charts/nvidia-installer/templates/ds-k8s-nvidia-driver.yaml]: Same as for the other image property.

File: charts/nvidia-installer/templates/ds-k8s-nvidia-deviceplugin.yaml (hunk 1 above)
Comment (thomasjungblut) [reply charts/nvidia-installer/templates/ds-k8s-nvidia-deviceplugin.yaml]: done

File: charts/nvidia-installer/templates/ds-k8s-nvidia-driver.yaml (hunk 2 above)
Comment (thomasjungblut) [reply charts/nvidia-installer/templates/ds-k8s-nvidia-driver.yaml]: done

**Review by afritzler — APPROVED** [review]
/lgtm

**Pull request merged**